lib/updater/synchronize.rb

```diff
@@ -74,6 +74,8 @@ def dependencies
         pods_dependencies << @config.pods
         
         pods_dependencies.flatten!.uniq!
+
+        pods_dependencies.reject! { |dependency| @config.exclude.include? dependency }
```

Comment:
I'm willing to change the name of the new configuration option if something is suggested :wink: